src/main/java/com/bankofspring/hello/HelloWorldImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.bankofspring.hello;
+
+public class HelloWorldImpl implements GreetingService {
+
+	private String greeting = "Hello, world!";
+	
+	public HelloWorldImpl() {
+		super();
```

Comment:
I don't really think this is required since this is already implicitly called. You only really need a super call, if you don't have a default constructor in the superclass or you want to explicitly use a different superclass constructor. But meh.